src/storage/src/table/cell_based_table.rs

```diff
@@ -69,6 +69,13 @@ fn err(rw: impl Into<RwError>) -> StorageError {
     StorageError::CellBasedTable(rw.into())
 }
 
+pub enum Op {
+    Insert,
+    Delete,
+    UpdateDelete,
+    UpdateInsert,
+}
```

Comment:
suggest use the `enum Op` in `common::array::stream_chunk`

Reply:
Sounds good.